Git lfs files still missing

atwilso opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

atwilso committed:
Add check for LFS smudge files

During the build, check for LFS smudge files in the repository and
error out if they're found.

atwilso committed:
Bump build number in meta.yaml

Need a new build to clobber smudge files.

atwilso commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-forge-tick (the bot)`. Your recipe  may not receive automatic updates and/or may not be compatible with conda-forge's infrastructure. Please check the logs for more information and ensure your recipe can be parsed.
 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/16393167765](https://github.com/conda-forge/conda-forge-webservices/actions/runs/16393167765). Examine the logs at this URL for more detail.</sub>

Commit:
MNT: Re-rendered with conda-build 25.5.0, conda-smithy 3.51.1, and conda-forge-pinning 2025.07.19.13.47.00

atwilso committed:
Run find in appropriate directory

Find needs to run in $SRC_DIR/tracktable-data and ignore the
.git subdirectory.

atwilso committed:
Only run du on checked-out copy of repository

This way we will not be misled by LFS objects in the .git subdirectory.

atwilso committed:
Add `git lfs pull` to build

Apparently it's not enough to run `git lfs fetch` -- I also need
`git lfs pull` to rehydrate the files.

atwilso committed:
Fix path to checked-out copy

I apparently had the path wrong.  If this fails too I'll add an
ls -R and a couple more print statements.

atwilso committed:
Diagnostic information for the DU and find

Where are we and what's here?

atwilso committed:
Add SRC_DIR to diagnostic information

atwilso committed:
Comment out ls -R

I think we've gotten what we need from the ls -R statement.

atwilso committed:
Add else clause after find statement for smudge files

Make sure we print an OK message to reassure the poor developer

Why are we getting that error exit from find?

atwilso committed:
Comment out smudge check

The command is causing errors and we get what we need from the
`du` statement, so comment it out for now.
